src/page-modules/contact/components/form/form.module.css

```diff
@@ -18,6 +18,10 @@
   gap: var(--spacings-small);
 }
 
+.label_disabled {
+  color: var(--text-colors-disabled);
+}
+
```

Comment:
Could this also be done with `: disabled` selector as you have done with cursor?

Reply:
I dont think so. Is that possible when the <label> is outside <input/>?

Reply:
Oh, it's the label.